client/src/apiCalls.js

```diff
@@ -25,6 +25,22 @@ const makeRequest = async (subdom, callMethod, header, data, onSucess, onError)
     .catch(error => onError(error))
 }
 
+const makeGETRequest = async (subdom, header, onSucess, onError) => {
```

Comment:
I made a similar function for GET request in another PR.
We should perhaps need to remove the redundancy once everything is merged to dev. Just to avoid any confusions in the future.